unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -356,7 +356,15 @@ public async Task LoadScene(CancellationToken token, int sceneIndex = -1, bool s
                 else
                 {
                     PerformanceAnalytics.GLTFTracker.TrackFailed();
-                    throw; 
+
```

Comment:
I believe all this is to fix the GLTF Importation.

It is fine but for the future, if time contrains allow, let's try to put it on its own PR.

Reply:
It is fixed in another PR, but I merged that pr in here to be able to test